visual-dotnet/SauceLabs.Visual/GraphQL/RegionIn.cs

```diff
@@ -0,0 +1,48 @@
+using Newtonsoft.Json;
+using OpenQA.Selenium;
+using SauceLabs.Visual.Models;
+
+namespace SauceLabs.Visual.GraphQL
+{
+    internal class RegionIn
+    {
+        [JsonProperty("name")]
+        public string? Name { get; }
+        [JsonProperty("x")]
+        public int X { get; }
+        [JsonProperty("y")]
+        public int Y { get; }
+        [JsonProperty("width")]
+        public int Width { get; }
+        [JsonProperty("height")]
+        public int Height { get; }
+
+        public RegionIn(int x, int y, int width, int height)
+        {
+            X = x;
+            Y = y;
+            Width = width;
+            Height = height;
+        }
+        public RegionIn(string name, int x, int y, int width, int height) : this(x, y, width, height)
+        {
+            Name = name;
+        }
+
+        public RegionIn(IWebElement input)
+        {
+            X = input.Location.X;
+            Y = input.Location.Y;
+            Width = input.Size.Width;
+            Height = input.Size.Height;
+        }
+
+        public RegionIn(IgnoreRegion input)
+        {
+            X = input.X;
+            Y = input.Y;
+            Width = input.Width;
+            Height = input.Height;
+        }
```

Comment:
small nit: 
if something needs to change in the initialization logic (lets say X must be always greater than 0) then we have one code path to do it.
```suggestion
        public RegionIn(IWebElement input) : this(input.X, input.Y, input.Width, input.Height)
        { }

        public RegionIn(IgnoreRegion input): this(input.X, input.Y, input.Width, input.Height)
        { }
```